CODEOWNERS

```diff
@@ -0,0 +1,15 @@
+# code owners are automatically assigned to review PRs
+
+# The goal is to not have more than 2 people responsible to avoid having too
+# many reviewers which often leads to nobody feeling responsible to actually
+# perform the review.
+
+dependencies/non-integer/ @mgudemann
+docs/delegation_design_spec/ @dnadales @kantp
+hs/ @mgudemann @JaredCorduan
+latex/ @JaredCorduan @mgudemann
+nix/ @nc6
+scripts/ @nc6
```

Comment:
For `nix` and `scripts` I guess @ruhatch could be a second code owner. If that's not possible I volunteer to be the second owner, however whenever I'm doing something with `nix` most of the stuff feels like black magic to me :)

Reply:
Yeah, I'm happy to help with `nix` related things

Reply:
You can add me to `nix` and `scripts` I think